start.sh

```diff
@@ -13,7 +13,7 @@ echo -en "\n\nWaitingForVonWebserver"; RES=""; while [[ -z "$RES" ]]; do sleep .
 # START F2P ARIES CLOUD AGENT
 docker-compose -f ./docker/docker-compose.yaml -p veritable-demo up --build -d
 echo -en "\n\nWaitingForIssuer"; RES=""; while [[ -z "$RES" ]]; do sleep .1; RES=$(curl -sf localhost:8021/status 2>&1); echo -n .; done
-echo -en "\n\nWaitingForHolder"; RES=""; while [[ -z "$RES" ]]; do sleep .1; RES=$(curl -sf localhost:8031/status 2>&1); echo -n .; done
+echo -en "\n\nWaitingForHolder"; RES=""; while [[ -z "$RES" ]]; do sleep .1; RES=$(curl -sf localhost:8031/health 2>&1); echo -n .; done
```

Comment:
so is it just a holder that has health check endpoint?

Reply:
yes because it's hitting the proxy not the actual aca-py